tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Execute a Runtime Upgrade
+description: Learn how to perform forkless runtime upgrades on a Polkadot SDK-based blockchain. Add features and modify parameters without network disruption.
+---
+
+# Execute a Runtime Upgrade
+
+## Introduction
+
+In the previous tutorial, you learned how to add pallets to the runtime and spin up your custom chain locally. Building upon that knowledge, this tutorial takes you a step further by demonstrating how to upgrade a runtime in a live, running network without interrupting its operations.
```

Comment:
Same comment here about the previous tutorials situation. Please check all of the guides and make sure we're not assuming that the user knows what the previous tutorials are